build/common.sh

```diff
@@ -576,20 +545,39 @@ function kube::build::ensure_data_container() {
     local -ra docker_cmd=(
       "${DOCKER[@]}" run
       --volume "${REMOTE_OUTPUT_GOPATH}"
+      --volume /usr/local/go/pkg/linux_386_cgo
```

Comment:
Added a note. I didn't see an easy way to automate it.  The way that `--fastbuild` is implemented would mean that we'd have to delete/recreate the data container or refactor that.